packages/elements/src/utils.ts

```diff
@@ -14,9 +14,12 @@ export const validateProps = (props: Record<string, any>, required: string[]) =>
 const CODE_RE = /[?&]code=[^&]+/;
 const STATE_RE = /[?&]state=[^&]+/;
 const ERROR_RE = /[?&]error=[^&]+/;
+const PARENT_RE = /[?&]photon-parent-auth=[^&]+/; // if this is present, these are auth params for a different Auth0 instance
```

Comment:
So we are requiring our customer to add `photon-parent-auth`, can't we just do the inverse and add a `photon=true` flag so our customers don't have to do anything?

Reply:
@sam So If I understand correctly:
- we programmatically append `photon=true` to the redirect uri passed into the `<photon-client>`
- in Photon client, we would check for the presence of `photon=true`
- on their side they would still need to add `skipRedirectCallback={window.location.href.includes('photon=true')}`